clients/web/src/views/widgets/AccessWidget.js

```diff
@@ -133,11 +160,55 @@ var AccessWidget = View.extend({
     },
 
     _makeTooltips: function () {
-        this.$('.g-action-remove-access').tooltip({
+        this.$('.g-access-action-container a,.g-tooltip').tooltip({
             placement: 'bottom',
             animation: false,
             delay: {show: 100}
         });
+
+        this.$('.g-flag-label span').tooltip({
+            placement: 'bottom'
+        });
+
+        // Re-binding popovers actually breaks them, so we make sure to
+        // only bind ones that aren't already bound.
+        _.each(this.$('.g-action-manage-flags'), el => {
+            if (!$(el).data('bs.popover')) {
+                $(el).popover({
+                    trigger: 'manual',
+                    html: true,
+                    placement: 'left',
+                    viewport: {
+                        selector: 'body',
+                        padding: 10
+                    },
+                    content: function () {
+                        return $(this).parent().find('.g-flags-popover-container').html();
+                    }
+                }).click(function () {
+                    $(this).popover('toggle');
+                });
+            }
+        });
+
+        _.each(this.$('.g-action-manage-public-flags'), el => {
```

Comment:
can you repeat that comment from above here, in case these two blocks get separated?